ontology/yaml/resources/subfields/subfields.yaml

```diff
@@ -79,7 +79,7 @@ descriptor:
   exchange: "the transfer of heat from one fluid stream to another."
   exercise: "Mode of running equipment to maintain functionality ('exercise mode')"
   exhaust: "Process or component used in the removal of air from a conditioned system to the outside atmosphere."
-  fault: "Fault of a device."
+  fault: "Fault of fire alarm system/panel e.g. ground fault, sensor fault, etc. A common of fire alarm system. It will not induce alarm."
```

Comment:
This definition is not clear. What is the difference between a fault and an alarm? Please clarify.

The definition should give me enough information for me to be able to use the keyword in a similar fashion elsewhere. I need to know how you are differentiating between faults and alarms.

Reply:
Also, if it is necessary to commit this subfield then it must be committed as a point type, not a descriptor. Please move.

Reply:
this is not sufficient differentiation between alarm and fault. what is a fault, specifically? what general rule can I use to differentiate it from an alarm?

Reply:
It is specifically applied to fire system only, and alarm point in fire alarm system is related to safety where fault is related to system.

Reply:
The description has been changed as follow:
 
fault: "Fault within a fire alarm system/panel e.g. ground fault, sensor fault, etc. It will not induce a fire alarm. For example, ground fault happens in the fire alarm system, it will not induce fire alarm but notify people it is a system fault"

Reply:
this fault description was removed